src/serial/remote/mod.rs

```diff
@@ -25,6 +25,22 @@ mod protocol_v3;
 mod protocol_v4;
 pub mod riscv_debug;
 
+pub struct RemoteCommands;
+
+#[allow(unused)]
+impl RemoteCommands
+{
+	/// This command asks the probe if the power is used
+	pub const GET_TARGET_POWER_STATE: &'static str = "!Gp#";
+	pub const JTAG_INIT: &'static str = "!JS#";
+	pub const NRST_GET: &'static str = "!Gz#";
+	pub const NRST_SET: &'static str = "!GZ#";
+	pub const PWR_GET: &'static str = "!Gp#";
```

Comment:
Duplicates the first constant, and the SET variant should be named the same way - ie `SET_TARGET_POWER_STATE`.

Reply:
Ah, good catch, removed the duplicate and renamed :)